### ISSUE TITLE
Class switching in class selection needs 2 clicks instead of 1

### ISSUE THREAD

**Issue by GingerAdonis** [JarnoVgr/ZombieSurvival#11]
When selecting a class and then clicking on another doesn't work as expected. First you need to unselect an option and then select a different one.

**Comment by Dubyx** [comment]
Hmm could be an easy change or complex change, either way it should improve the smoothness of the mod.

**Comment by Dubyx** [comment]
This is going to be very complex and a lot of time/re-coding will be required. So I expect this will be put on the back burner after the full release and be one of the updates which follows after.

**Comment by GingerAdonis** [comment]
It's the first screen people see. I think this should be done before release. Even if it's just a quick 'to make it ok for release' change

**Comment by Dubyx** [comment]
Its going to be a right pain in the backside, and due to my time restrictions this alone could put us back 2 plus weeks. But if you think you could take a look at it, or give some pointers then pworth.lua is the file in question.

**Comment by Dubyx** [comment]
This won't be able to be resolved by June for the Release date. Its going to take a very long time of thinking how to re-code how Jetboom did his menu selection. As he added the whole point 'worth' system . So its going to take me a month of two to get round to looking at it. Its not game breaking and there will be more important things to deal with. 

So this will be a low priority fix for now, and then a high priority fix after release.

**Comment by Dubyx** [comment]
Cevadar has fixed this issue and will create a Pull request for the fix. It will be accepted if I'm happy with the code and the way it operates etc...

**Comment by Dubyx** [comment]
fixed
